src/print.rs

```diff
@@ -7,7 +7,8 @@ use crate::context::Context;
 use crate::module::Module;
 use crate::modules;
 
-const PROMPT_ORDER: &[&str] = &[
+//List of all prompt order
+const ALL_PROMPT_LIST: &[&str] = &[
```

Comment:
I think this might be a little more descriptive.
```suggestion
const MODULE_LIST: &[&str] = &[
```